packages/protocol/src/debug.ts

```diff
@@ -0,0 +1,306 @@
+/**
```

Comment:
What's the difference with `packages/playwright-core/src/protocol/debug.ts` and `packages/protocol/src/metainfo.ts`?